### DISCUSSION TITLE
in Farming, but No any farmer_server message could be found  in debug.log ,is it normal ? what could i do ?

### DISCUSSION THREAD

**Discussion by xlkjmjp6587** [Chia-Network/chia-blockchain#6739] [Support]
my chia had been going well  , but after some adjustment to our main server of DEBIAN,  the chia seems work not so well for three days,  
the summary are these i show below:
...... chia farm summary
Farming status: Farming
Total chia farmed: 138.0
User transaction fees: 0.0
Block rewards: 138.0
Last height farmed: 420737
Plot count: 482
Total size of plots: 54.009 TiB
Estimated network space: 23238.574 PiB
Expected time to win: 3 months

the summary looks so good,  but it win nothing ,we got no reward.    the most strange thing is that i can not find any message logged by 'farmer_server'  in log file .../log/debug.log*;     even if i check the files with "cat ../log/debug.log* |  grep 
 'farmer_server'    ,i still got nothing  about farmer_server.    

      are those farmer_server messages from chia_farmer ?   
       what is the problem?  somebody tell me what i could do ......

**Comment by loppefaaret** [comment]
to see farmer messages in the `debug.log` - you need to set DEBUG level logging, and restart the farmer

**Reply by xlkjmjp6587** [reply]
i solved this problem by the way you provided  last night , thanks for your help~~

**Comment by BrandtH22** [comment]
@xlkjmjp6587 , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
